src/components/LoadingSkeletons/BarChartLoadingSkeleton.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FunctionComponent } from 'react';
+import LoadingMessage from '../LoadingMessage/LoadingMessage';
+import { backgroundGray } from './LoadingSkeletonsSvgUtil';
+import LoadingSkeleton, {
+	ILoadingSkeletonProps,
+	IStandardSkeleton,
+} from './LoadingSkeleton';
+
+export const BarChartSkeleton: FunctionComponent<IStandardSkeleton> = props => {
+	const { width, height, className } = { ...props };
```

Comment:
Take a look at Badge or Button to see the standard syntax for props in Lucid. 

`export const BarChartSkeleton = (props: IBarChartSkeletonProps): React.ReactElement => {

  const {
      width,
      height,
      className
      } = props;
`

Reply:
Done